### ISSUE TITLE
Midterm Review (ky276)

### ISSUE THREAD

**Issue by kaysieyu** [mnercessian/ORIE4741proj#10]
This project aims to predict the outcome and different point differentials of NFL games. 

Things I Liked
1. The team has a really great dataset to work with. They did a good job cleaning the data and identifying an initial set of features that could help them with their analysis. There are a lot of potential features and data they can use to produce and test the effectiveness of their models. 
2. Good job with the preliminary investigation. Fitting a least squares regression is always a nice starting point and it produced relatively high prediction accuracy for both the training and test sets (58.5% and 64.6% accuracy respectively). 
3. There is a clear plan moving forward for next steps and ways to further the analysis. The use of loss functions and regularizers may yield better predictions that least squares. 

Areas of Improvement
1. I would have liked to know how the team dropped columns and how they decided if certain data was worth keeping or removing. Was there specific criteria? Was too much data missing from the columns that were dropped?
2. There is mention that the scatter point appeared to be noisy and that the error vales seemed high for the true vs predicted point differentials. I wonder if training your model on more features (not just the home and away team points per game) would improve the model and reduce the error.
3. It could be interesting in your final write up to include your findings on what features have the highest influence or impact on the models you create. Would previous point history or injuries be key indicators of success or are other miscellaneous factors more influential?

Overall, I think this is a good start and I look forward to seeing what the team comes up with for the final paper.
